src/test/java/epam/training/hw3/ex2/DifferentElementsTest.java

```diff
@@ -0,0 +1,113 @@
+                .isTrue();
+        softAssertions
+                .assertThat(homePage.getHeader().getName())
+                .isEqualTo(EXPECTED_USER_NAME);
+
+
+        //5. Open through the header menu Service -> Different Elements Page
+        DifferentElementsPage differentElementsPage = homePage.getHeader().clickDifferentElement();
+        softAssertions
+                .assertThat(differentElementsPage.getURL())
+                .isEqualTo(properties.getString("diffURL"));
+
+        //6. Select checkboxes
+        CheckBox checkBoxWater =  differentElementsPage.getDifferentElements().getCheckBox(CHECKBOX_TEXT.get(0));
+        CheckBox checkBoxWind = differentElementsPage.getDifferentElements().getCheckBox(CHECKBOX_TEXT.get(2));
+
+        softAssertions
+                .assertThat(checkBoxWater)
+                .isNotNull();
+        softAssertions
+                .assertThat(checkBoxWind)
+                .isNotNull();
+
+        checkBoxWater.click();
+        checkBoxWind.click();
+
+        softAssertions
+                .assertThat(checkBoxWater.isSelected())
+                .isTrue();
+        softAssertions
+                .assertThat(checkBoxWind.isSelected())
+                .isTrue();
+
+        //7. Select radio
+        Radio radio = differentElementsPage.getDifferentElements().getRadio(RADIO_TEXT.get(3));
+        softAssertions
+                .assertThat(radio)
+                .isNotNull();
+        radio.click();
+        softAssertions
+                .assertThat(radio.isSelected())
+                .isTrue();
+
+        //8. Select in dropdown
+        Select dropdown = differentElementsPage.getDifferentElements().getDropDown(DROPDOWN_TEXT.get(3));
+        softAssertions
+                .assertThat(dropdown)
+                .isNotNull();
+        softAssertions
+                .assertThat(dropdown.getFirstSelectedOption().getText())
+                .isEqualTo(DROPDOWN_TEXT.get(3));
+
+        //9. Assert that for each checkbox there is an individual log row
+        // and value is corresponded to the status of checkbox.
+
+        //   Assert that for radiobutton there is a logList row
+        //   and value is corresponded to the status of radiobutton.
+
+        //   Assert that for dropdown there is a logList row
+        //   and value is corresponded to the selected value.
+        List<String> expectedLog = new ArrayList<>(
+                List.of(
+                        CHECKBOX_LOG + CHECKBOX_TEXT.get(0),
+                        CHECKBOX_LOG + CHECKBOX_TEXT.get(2),
+                        RADIO_LOG + RADIO_TEXT.get(3),
+                        DROPDOWN_LOG + DROPDOWN_TEXT.get(2)
+                ));
+
+        expectedLog.forEach(expLog ->
+                softAssertions
+                        .assertThat(differentElementsPage.getLogPanel().findLogs(expLog))
+                        .isNotNull()
+        );
+
+        //AssertAll
+        softAssertions.assertAll();
```

Comment:
Better to move `softAssertions.assertAll()` to tear down method (with the annotation `@AfterClass` or `@AfterTest`). Because if the test fails before `assertAll()` we won't see any assertions

Reply:
Moved softAssertions.assertAll() to tear down method